tests/encryption_/models.py

```diff
@@ -26,26 +26,27 @@
 from django_mongodb_backend.models import EmbeddedModel
 
 
-class Billing(EmbeddedModel):
-    cc_type = models.CharField(max_length=50)
-    cc_number = models.CharField(max_length=20)
-
-
-class PatientRecord(EmbeddedModel):
-    ssn = EncryptedCharField(max_length=11, queries={"queryType": "equality"})
-    billing = EncryptedEmbeddedModelField(Billing)
-
-
 class Patient(models.Model):
```

Comment:
Doesn't Patient also need EncryptedModelBase?

Reply:
Looks like none of them needed it! 6675258ce151b8885b32c7f7c6ebf4e23f0cfda2

Reply:
It's needed when `Features.supports_queryable_encryption = False`, otherwise the test suite initialization fails with an error like  "Encrypted collections are not supported on standalone."